_api_app/app/Sites/multisites.twig

```diff
@@ -1,29 +1,14 @@
-{% if multisitesOption|length  > 1 %}
-  <ul id="multisites">
-    {% for multisitesOption in siteName => site %}
-      {% if environment == 'engine' || multisiteOption != siteName || ( siteName =='0' && multisiteOption !='' ) %}
-        <li {% if multisiteOption === siteName || ( siteName =='0' && multisiteOption =='' ) %} class="selected" {% endif %}>
-          <a href="{{ bertaLink site = siteName }}">
-            {% if site['title']['value'] !='' %}
-              {{ site['title']['value'] }}
-            {% else %}
-              {% if siteName =='0' %}
-                Main Site
-              {% else %}
-                {{ siteName }}
-              {% endif %}
-            {% endif %}
+<ul id="multisites">
+  {% for sites in multisite %}
```

Comment:
Don't mess the singular vs plural.
`for site in multisites`
or even better
`for site in sites`